home/models.py

```diff
@@ -15,3 +15,35 @@ class Student(models.Model):
     social_url = models.CharField(max_length=500, null=True, blank=True)
     is_active = models.CharField(max_length=500, null=True, blank=True)
     normalized_name = models.CharField(max_length=500, null=True, blank=True)
+
+    subject = models.ForeignKey('home.Subject',
+                                on_delete=models.SET_NULL,
+                                null=True)
+    book = models.OneToOneField('home.Book',
+                                on_delete=models.CASCADE,
+                                null=True)
+    teacher = models.ForeignKey('home.Teacher',
```

Comment:
maybe better many to many?
cause one teacher can has many students and one student can has many teachers? or i wrong?

Reply:
Teacher has many students.
Select students where teacher=id